index.js

```diff
@@ -11,4 +11,7 @@ module.exports = {
     },
     delimiter: require('./lib/extension/delimiter.extension'),
     listMPU: require('./lib/extension/listMPU.extension'),
+    Testing: {
```

Comment:
why not just testingMatrix instead of nesting?  Are there going to be other items exported under this heading?  Even if so, let's keep the lowercase convention.